packages/test/src/test-otel.ts

```diff
@@ -129,6 +153,59 @@ test.serial('Exporting OTEL metrics from Core works', async (t) => {
   }
 });
 
+test.serial('Exporting OTEL metrics using OTLP/HTTP from Core works', async (t) => {
+  let resolveCapturedRequest = (_req: http.IncomingMessage) => undefined as void;
+  const capturedRequest = new Promise<http.IncomingMessage>((r) => (resolveCapturedRequest = r));
```

Comment:
You wrote it so great, but, IMO for things like this it's fine to just ensure the option makes it to Core since Core went through the rigamarole of having an actual server etc